x-pack/plugin/esql/qa/testFixtures/src/main/resources/math.csv-spec

```diff
@@ -335,7 +335,7 @@ ROW base = -4, exponent = 0.5
 // end::powNeg-sqrt[]
 ;
 warning:Line 2:12: evaluation of [POW(base, exponent)] failed, treating result as null. Only first 20 failures recorded.
-warning:Line 2:12: java.lang.ArithmeticException: invalid result: pow(-4.0, 0.5)
+warning:Line 2:12: java.lang.ArithmeticException: invalid result when computing pow
```

Comment:
I presume it is considered a security concern to include parts of the query in the error message? Pity, as this is so much nicer for debugging.

Reply:
It's also very hard to test.  The tests all now inject many values randomly into the execution, and we didn't build a mechanism to also inject those into the warning messages.  We could solve that with some wild cards or such though.

I agree it aids in debugging, and I am also sad to lose it, but it really feels very risky.  If you want to go to infosec and get them to sign off on this type of error message, I'm happy to build test support for doing it and put them back, but absent an affirmative consent from infosec I really think it's a bad idea.

Reply:
Just trying to understand the concern, if the user would simply do a `| KEEP base, exponent`, that'd reveal anyways the values, provided they have the appropriate permissions, no? They can only apply operations on fields they're entitled to.
In any case, we are providing explicit failure messages in most of other places (like numerical overflows, conversions etc.) already. I think it might make sense to either "fix" them all, or keep being explicit for now and address these security concerns holistically.